Plugin.cs

```diff
@@ -53,6 +55,17 @@ private void LoadConfig(Config conf)
                 Log.Debug("Adding initial blank preset");
                 PluginConfig.Instance.ExtraColorPresetsV2.Add(new ExtraColorPresetV2());
             }
+            
+            ReinitUniqueSelectables();
+        }
+
+        public static void ReinitUniqueSelectables()
+        {
+            ExtraColorPresetsUniqueSelectable = new List<string>();
+            PluginConfig.Instance.ExtraColorPresetsV2.ForEach(scheme =>
+            {
+                ExtraColorPresetsUniqueSelectable.Add(scheme.colorSchemeId);
+            });
```

Comment:
This can be simplified:
```suggestion
            ExtraColorPresetsUniqueSelectable = PluginConfig.Instance.ExtraColorPresetsV2.ConvertAll(scheme => scheme.colorSchemeId);
```